src/linagora.esn.calendar/app/planning/calendar-planning.less

```diff
@@ -47,4 +47,23 @@
   .fc-list-item-title {
     font-weight: 500;
   }
-}
\ No newline at end of file
+
+  .planning-event-accepted {
+  }
+
+  .planning-event-needs-action {
+    opacity: @cal-event-fade;
+  }
+
+  .planning-event-tentative {
+    i {
+      font-size: 1.2em;
+      margin-right: 3px;
+    }
+  }
+
+  .planning-event-declined {
+    opacity: @cal-event-fade;
+    text-decoration: line-through !important;
```

Comment:
Why do we need to have `important` here?

If it is mandatory, `important` require a comment to explain why?